.evergreen/config.yml

```diff
@@ -145,7 +145,7 @@ functions:
             # If this was a patch build, doing a fresh clone would not actually test the patch
             cp -R ${PROJECT_DIRECTORY}/ $DRIVERS_TOOLS
           else
-            git clone https://github.com/mongodb-labs/drivers-evergreen-tools.git $DRIVERS_TOOLS
+            git clone --branch GODRIVER-3148 https://github.com/blink1073/drivers-evergreen-tools.git $DRIVERS_TOOLS
```

Comment:
It seems safe to update this line since the branch has been merged.

Reply:
We still need this PR to be merged: https://github.com/mongodb-labs/drivers-evergreen-tools/pull/414